src/main/java/core/mate/academy/service/impl/TruckProducerImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.mate.academy.service.impl;
+
+import core.mate.academy.model.Truck;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducerImpl implements MachineProducer<Truck> {
```

Comment:
The `TruckProducerImpl` class should not start with an empty line. Remove the empty line to follow best practices.